packages/webdriverio/src/utils.ts

```diff
@@ -130,13 +130,13 @@ async function assertFrameReady(client: Browser): Promise<void> {
 }
 
 export const axeRunPartial = (
-  client: Browser,
+  client: WdioBrowser,
   context?: SerialContextObject,
   options?: RunOptions
 ): Promise<PartialResult> => {
   return promisify(
-    client
-      .executeAsync<string, []>(
```

Comment:
Older versions of webdriverio [don't declare the type](https://github.com/webdriverio/webdriverio/blob/v6.12.1/packages/webdriverio/src/commands/browser/executeAsync.ts#L46) of the return so we can't declare it either. This defaults to `<unknown, []>` for the type in newer versions which is why we need to make the change below to make the string type `unknown` and then cast it to a `string`